use-cases/customer_churn/0_cust_churn_overview_dw.ipynb

```diff
@@ -0,0 +1,1651 @@
+{
```

Comment:
<p>change % to !</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2071/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>